controllers/managedocs_controller.go

```diff
@@ -126,6 +128,9 @@ type ManagedOCSReconciler struct {
 	AlertSMTPFrom                string
 	CustomerNotificationHTMLPath string
 	DeploymentType               string
+	RhobsSecretName              string
+	RhobsApiUrl                  string
+	RhSSOTokenUrl                string
```

Comment:
See naming convention and terminology
```suggestion
	RHOBSSecretName              string
	RHOBSEndpoint                  string
	RHSSOTokenEndpoint                string
```

Reply:
Done